maven-repository-manager/pom.xml

```diff
@@ -47,6 +47,10 @@
       <artifactId>spi</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.jboss.logging</groupId>
+      <artifactId>jboss-logging</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
Okay, I've moved it to test scope and verified that the tests succeed (in this module, at least). I'm still having trouble with later modules in the build that I believe are unrelated (they haven't ever worked on this machine).